tsconfig.json

```diff
@@ -4,8 +4,8 @@
     "esModuleInterop": true,
     "allowSyntheticDefaultImports": true,
     "jsx": "react",
-    "lib": ["dom", "es2015", "es2017", "es2018", "es2019", "es2020", "es5", "es6", "es7", "esnext"],
-    "module": "es2020",
+    "lib": ["dom", "es2015", "es2017", "es2018", "es2019", "es5", "es6", "es7", "esnext"],
```

Comment:
Did you mean to remove this? Should tidy up the comment if so - be wary, may have effects on other packages!